src/cascadia/TerminalSettingsEditor/Appearances.cpp

```diff
@@ -526,7 +618,7 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 
         // find one axis that does not already exist, and add that
         // if there are no more possible axes to add, the button is disabled so there shouldn't be a way to get here
-        const auto possibleAxesTagsAndNames = ProfileViewModel::FindFontWithLocalizedName(FontFace()).FontAxesTagsAndNames();
+        const auto possibleAxesTagsAndNames = ProfileViewModel::FindFontWithLocalizedName(_primaryFontName).FontAxesTagsAndNames();
```

Comment:
These are all placeholders until we refactor the font axis/feature code later on.
Refactoring it as part of this PR was not possible unfortunately.

Reply:
Just curious as to what the refactor entails - do we need to combine all the possible axes/features for all the fonts in the given family?

Reply:
Yes. We'll have to basically detach the `Font` object entirely from axes and features. In my opinion we should move the (re)generation of axes and features from `Font`, `ProfileViewModel`, and `Appearance` all over into `AppearanceViewModel`.
There, we have access to the parsed list of fonts families (which this PR adds) and this allows us to generate the union of axes and features. Then we can split the collection up into those that are currently in use and those that are still available for selection and expose it to the view.

Reply:
Also, it may be worth mentioning that we create `AppearanceViewModel`s eagerly when loading the settings UI, which is not great, because this causes all the DirectWrite code to run eagerly too. It'd be nice if they were created lazily somehow.